alexey.zhuravlov/task1/findDuplicates/src/com/company/FindDuplicates.java

```diff
@@ -0,0 +1,56 @@
+package com.company;
+
+import java.util.*;
+
+public class FindDuplicates
+{
+
+  public static List<BankAccount> findDuplicates1(Collection<BankAccount> collA, Collection<BankAccount> collB)
+  {
+    Set<BankAccount> collASet = new HashSet<>(collA);
+    collASet.retainAll(collB);
+    return new ArrayList<BankAccount>(collASet);
+  }
+
+
+  /*public static List<BankAccount> findDuplicates2(Collection<BankAccount> collA, Collection<BankAccount> collB)
+  {
```

Comment:
Why this code is commented? Please dont leave commented code inside pull request